### PR TITLE
Added Homepage

### PR DESCRIPTION
**Pull request by Ankur9669**
image

### CODE REVIEW

File: common.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+}
+.input-box .nav-searchbar{
+   color: var(--primary-text-color);
+}
+.input-box .nav-searchbar:focus{
+    border: 2px solid var(--primary-color);
+}
+.page-container{
+    display: grid;
+    grid-template-columns: 300px auto;
```
Comment (sri-rishi) [inline common.css]: can use rem here instead of px

File: common.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    border: 2px solid var(--primary-color);
+    border-radius: 2rem;
+}
+.active{
+    background-color: var(--primary-color);
+    border-radius: 2rem;
+}
+.nav-item-mobile i{
+    flex: 0.2;
+    margin: 0rem 0.5rem 0rem 2.5rem;
```
Comment (sri-rishi) [inline common.css]: Don't use 0rem or 0px just using 0 will work

**Review by sri-rishi — CHANGES_REQUESTED** [review]
You hamburger icon is not showing apart form changes mentioned code  and UI looks good

File: index.html [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+                  <i class="fa fa-flag" aria-hidden="true"></i>
+                </span>
+                <div class="progress-item-text-container">
+                  <p class="font-large weight-semi-bold primary-text-color">
+                    0
+                  </p>
+                  <p
+                    class="font-extra-small weight-semi-bold primary-text-color"
+                  >
+                    Challenge Completed
```
Comment (vanshitamoolchandani) [inline index.html]: Just a suggestion as all these are heading you can use heading tags instead of p tag.
Other than this looks good to merge.

Comment (Ankur9669) [reply index.html]: That's a nice catch i will do it right away

File: common.css (hunk 2 above)
Comment (Ankur9669) [reply common.css]: Done

File: common.css (hunk 1 above)
Comment (Ankur9669) [reply common.css]: Done

**Review by sri-rishi — APPROVED** [review]
changes looking good to merge

**Pull request merged**